README.md

```diff
@@ -1,91 +1,37 @@
 # finch-tensor-lite
 
-The pure-python rewrite of [Finch.jl](https://github.com/finch-tensor/Finch.jl) to eventually replace [finch-tensor](https://pypi.org/project/finch-tensor/)
+A pure-Python rewrite of [Finch.jl](https://github.com/finch-tensor/Finch.jl), designed to eventually replace [finch-tensor](https://pypi.org/project/finch-tensor/).
 
-## Source
+## Features
 
-The source code for `finch-tensor-lite` is available on GitHub at [https://github.com/finch-tensor/finch-tensor-lite](https://github.com/FinchTensor/finch-tensor-lite)
+- **Write Once, Run Efficiently:** Express tensor computations in a clear, NumPy-like style—`finch-tensor-lite` automatically optimizes and fuses operations for both dense and sparse data.
+- **Flexible Execution:** Effortlessly switch between eager (immediate) and lazy (deferred, fused) evaluation to balance simplicity and performance.
+- **Advanced Optimization:** Benefit from algebraic and symbolic rewrites, constant folding, and other compiler-level optimizations—no manual tuning required.
+- **Rich Array Support:** Work seamlessly with dense, sparse, and structured arrays, including custom background (zero) values and user-defined formats.
+- **Interoperable:** Integrates smoothly with NumPy and the broader Python scientific ecosystem.
+- **Modern API:** Follows the [Array API specification](https://data-apis.org/array-api/latest/) for consistency and compatibility.
 
 ## Installation
 
-`finch-tensor-lite` is available on PyPi, and can be installed with pip:
+Install from PyPI:
 ```bash
 pip install finch-tensor-lite
 ```
 
-## Contributing
-
-### Code Of Conduct
-
-We adhere to the [Python Code of Conduct](https://policies.python.org/python.org/code-of-conduct/)
-
-### Collaboration Practices
-
-For those who are new to the process of contributing code, welcome! We value your contribution, and are excited to work with you. GitHub's [pull request guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) will walk you through how to file a PR.
-
-Please follow the [SciML Collaborative Practices](https://docs.sciml.ai/ColPrac/stable/) and [Github Collaborative Practices](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/helping-others-review-your-changes) guides to help make your PR easier to review.
-
-In this repo, please use the convention <initials>/<branch-name> for pull request branch names, e.g. ms/scheduler-pass.
-This way in bash when you type your initials git checkout ms/ and <tab> you can see all your branches. We will use other names for special purposes.
-
-### Packaging
-
-Finch uses [poetry](https://python-poetry.org/) for packaging.
-
-To install for development, clone the repository and run:
+For development (with tests and pre-commit hooks):
 ```bash
+git clone https://github.com/finch-tensor/finch-tensor-lite.git
+cd finch-tensor-lite
```

Comment:
It's kindof confusing to tell users to run pre-commit hooks. I think we need just the regular install instructions here, and dev install instructions go in contributing.md.